docs/installation.rst

```diff
@@ -114,6 +137,21 @@ Fedora/RHEL 8/CentOS 8
     $ sudo dnf install redhat-rpm-config gcc libffi-devel python3-devel \
         openssl-devel cargo
 
+RHEL 7/CentOS 7
```

Comment:
RHEL 8.3 and CentOS based on RHEL 8.3 have sufficently new Rust.
Rust in RHEL 8.2 is too old.